src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ConflictResolution.targets

```diff
@@ -25,6 +25,20 @@ Copyright (c) .NET Foundation. All rights reserved.
           To do this, we look at the files that would be copied local when CopyLocalLockFileAssemblies is true
           -->
       <_LockFileAssemblies Include="@(AllCopyLocalItems->WithMetadataValue('Type', 'assembly'))" />
```

Comment:
interesting, I guess the deps file is still correct since that gets copied from assets but this is something @eerhardt might hit if we tried to reconstruct the deps file from these items.